### PR TITLE
Feat: Add editorial place summary to plan details

### PR DESCRIPTION
**Pull request by timwangmusic**
## Description
Clearly describe changes in your PR, new feature or bug fixes.

## Solution
* Describe your code changes in detail for reviewers as well as documentation purposes. 
* For bug fixes, explain the solutions and how they fix the issues.
* For new features, describe the high-level ideas.

## Testing
- [x] Integration testing on Heroku staging
- [x] Added new unit tests or enhanced existing tests

## Checks
- [x] Have you removed commented code?
- [x] Have you used gofmt to format your code?

### CODE REVIEW

File: test/create_visit_location_test.go [hunk 1]
```diff
@@ -15,7 +15,7 @@ func TestCreatePlace(t *testing.T) {
 
 	expectedLatitude := 32.715736
 	expectedLongitude := -117.161087
-	place := POI.CreatePlace(name, microAddr, addr, "OPERATIONAL", "stay", nil, "landmark_mtv", 3, 4.5, "", nil, 0, expectedLatitude, expectedLongitude)
+	place := POI.CreatePlace(name, microAddr, addr, "OPERATIONAL", "stay", nil, "landmark_mtv", 3, 4.5, "", nil, 0, expectedLatitude, expectedLongitude, nil)
```
Comment (rwangsc18) [inline test/create_visit_location_test.go:18]: it's better to add some test for a place with valid summary info

**Review by rwangsc18 — COMMENTED** [review]
Overall LGTM. Left one comment about unit test coverage.

File: test/create_visit_location_test.go (hunk 1 above)
Comment (timwangmusic) [reply test/create_visit_location_test.go]: Done.

File: POI/places.go [hunk 2]
```diff
@@ -145,6 +146,14 @@ func (place *Place) GetUserRatingsTotal() int {
 	return place.UserRatingsTotal
 }
 
+func (place *Place) GetSummary() string {
+	return place.Summary
```
Comment (sananand007) [inline POI/places.go:150]: is it from here we are getting the piece of information 
```
Maurice A. Ferré Park
Spacious waterside park featuring a sand beach, large amphitheater & a playground.
```

**Review by sananand007 — APPROVED** [review]
Looks really nice 👍

**Pull request merged**